src/utility/hybridLock.h

```diff
@@ -35,6 +35,14 @@ class HybridLock {
 
   inline int load() const { return version_counter.load(); }
 
+  bool lockable() {
```

Comment:
Is this just for the testing purposes?

Reply:
Currently, this is only used for testing purposes. But since it could be used in other circumstances as well I added it directly to the hybridLock class.